lib/instrumentation/index.js

```diff
@@ -412,22 +419,26 @@ Instrumentation.prototype._startHook = function () {
 };
 
 Instrumentation.prototype._getPackageVersion = function (modName, modBaseDir) {
+  const modPath = path.join(modBaseDir, 'package.json');
+  if (this._versionsCache.has(modPath)) {
```

Comment:
Note for reviewer: both RITM and IITM hooks use this function and will get cached result if they request many times the version for the same `pacakge.json` file